### PR TITLE
Prompt for rh_registry_username if it is not provided

### PR DESCRIPTION
**Pull request by abaiken**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ install:
 # command to run tests
 script:
   - tox -e $TOX_ENV
+  - tox -e lint
```
Comment (myersCody) [inline .travis.yml:26]: Great catch Ash 🎃

File: qpc_tools/messages.py [hunk 2]
```diff
@@ -36,8 +36,8 @@
 SERVER_INSTALL_USERNAME_HELP = 'Set the server admin username (defaults to admin)'
 SERVER_INSTALL_PASSWORD_HELP = 'Set the server admin password. If not provided, '\
     'user will be prompted.'
-SERVER_INSTALL_REGISTRY_UN_HELP = 'Set the redhat.registry.com username'
-SERVER_INSTALL_REGISTRY_PASS_HELP = 'Set the rehat.registry.com password. '\
+SERVER_INSTALL_REGISTRY_UN_HELP = 'Set the registry.redhat.io username'
+SERVER_INSTALL_REGISTRY_PASS_HELP = 'Set the registry.redhat.io password. '\
```
Comment (myersCody) [inline qpc_tools/messages.py:39]: Another great catch ash 👍

File: .travis.yml (hunk 1 above)
Comment (kholdaway) [reply .travis.yml:26]: Thanks @abaiken

**Review by kholdaway — APPROVED** [review]

**Comment by kholdaway** [comment]
Nice job @abaiken!  Everything works great!

**Pull request merged**